config/initializers/sorcery.rb

```diff
@@ -0,0 +1,8 @@
+Rails.application.config.sorcery.submodules = []
+
+Rails.application.config.sorcery.configure do |config| 
+  # --- user config ---
+  config.user_config do |user|
```

Comment:
Unused block argument - `user`. You can omit the argument if you don't care about it.